app/src/main/java/de/westnordost/streetcomplete/quests/building_underground/IsBuildingUnderground.java

```diff
@@ -0,0 +1,51 @@
+package de.westnordost.streetcomplete.quests.building_underground;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class IsBuildingUnderground extends SimpleOverpassQuestType
+{
+	@Inject public IsBuildingUnderground(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "ways, relations with building and !location and layer~-.*";
+	}
+
+	public AbstractQuestAnswerFragment createForm()
+	{
+		return new YesNoQuestAnswerFragment();
+	}
+
+	public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes)
+	{
+		if(answer.getBoolean(YesNoQuestAnswerFragment.ANSWER)){
+			changes.add("location", "underground");
+		} else {
+			changes.add("location", "surface");
+		}
+	}
+
+	@Override public String getCommitMessage() { return "Add whatever building is fully underground"; }
+	@Override public int getIcon() { return R.drawable.ic_quest_building; }
```

Comment:
I made a special icon for this already a while back. If it is not already exported, you will find it in the quest icons svg

Reply:
It seems to not be exported. From looking at https://github.com/westnordost/StreetComplete/commit/429a92c932f95acb2356adce3965f66634760f21 it seems that exporting means saving .svg pattern (copied from larger file) to a new file with .xml extension.